vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -155,7 +155,8 @@
     <ClCompile Include="$(YogaDir)\yoga\YGNodePrint.cpp" DisableSpecificWarnings="4244;%(DisableSpecificWarnings)" />
     <ClCompile Include="$(YogaDir)\yoga\YGStyle.cpp" DisableSpecificWarnings="4244;%(DisableSpecificWarnings)" />
     <ClCompile Include="$(YogaDir)\yoga\YGValue.cpp" />
-    <ClCompile Include="$(YogaDir)\yoga\Yoga.cpp" DisableSpecificWarnings="4244;%(DisableSpecificWarnings)" />
+    <ClCompile Condition="'$(Platform)' == 'x64'" Include="$(YogaDir)\yoga\Yoga.cpp" DisableSpecificWarnings="4244;%(DisableSpecificWarnings)" FloatingPointModel="Strict" />
```

Comment:
Could you add a comment with a link to the issue here?

Reply:
Note that I'm only able to see a difference in we build x64 AND Release (x64 Debug looks ok)
![image](https://user-images.githubusercontent.com/22989529/79604345-c2d4ba80-80a2-11ea-895b-598d461ed1e3.png)